### ISSUE TITLE
FNX3-17120 ⁃ [Bug][a11y]  The 3 line menu(hamburger menu) cannot be selected on CNN.com while talkback is enabled

### ISSUE THREAD

**Issue by abodea** [mozilla-mobile/fenix#3570]
## Prerequisites
You have Talkback option enabled from your Android settings.

## Steps to reproduce
1. Launch Fenix.
1. Access **`www.cnn.com`**.
1. Select any element from the page.
1. Swipe on screen to the right for changing the Talkback position until reaching the top of the website where the `hamburger` menu is displayed.

### Expected behavior
The 3 line menu(hamburger menu) cannot be selected on `www.cnn.com` while having Talkback enabled.

### Actual behavior
The Talkback doesn't move to the 3 line menu.

### Device information
* Android device: **`Samsung Galaxy Note 9(Android 8.1.0).`**
* Fenix version: **`6/19 1.0.0-rc.2`** 
    1. Notes
* [Video](https://drive.google.com/file/d/1giV9slvexj8CzC-eb5jlmZkhJG7DSsUV/view?usp=sharing)
Please note that at the end of the video I selected the 3 line menu manually with the S-Pen to show that it can be accessible.

**Comment by ekager** [comment]
I was able to right swipe to the hamburger menu on latest master with talkback on. Adding QA needed label to retest :)

**Comment by softvision-miralobontiu** [comment]
Verified as fixed on the latest Nightly build with LG G7 Fit (Android 8.1), and Sony Xperia Z5 Premium (Android 7.1.1).
